cookbooks/cdo-github-access/recipes/default.rb

```diff
@@ -32,11 +54,10 @@
   group node[:current_user]
 end
 
-[
-  'config',
-  'id_rsa',
-  'id_rsa.pub',
-].each do |file|
+# Chef managed instances (staging, test, levelbuilder, production) have
+# the SSH key `ubuntu@code.org` Attribute from the Chef `baseline` Role. In rare cases, an adhoc can be provisioned
+# to be "Chef Managed" and will have the `id_rsa` and `id_rsa.pub` Chef Attributes.
```

Comment:
This comment is confusing; I don't know how to parse the first sentence, and I'm not sure what "in rare cases" means